pkg/services/featuremgmt/manager_test.go

```diff
@@ -0,0 +1,77 @@
+package featuremgmt
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
```

Comment:
```suggestion
	"github.com/stretchr/testify/assert"
```
Nit but I think this file should only use `assert` for the checks.